cloudbuild.yaml

```diff
@@ -17,23 +17,28 @@ steps:
     ln -s $$PWD $$shadow || stat $$shadow
 
     # Install ko from release.
-    curl -L -o ko.tar.gz https://github.com/google/ko/releases/download/v0.6.0/ko_0.6.0_Linux_i386.tar.gz
+    curl -L -o ko.tar.gz https://github.com/google/ko/releases/download/v0.8.2/ko_0.8.2_Linux_i386.tar.gz
     tar xvfz ko.tar.gz
     chmod +x ko
     alias ko=./ko
 
+    debug_tag=""
+    if [ "${TAG_NAME}" != "" ]; then
+      debug_tag=debug-${TAG_NAME}
```

Comment:
I think `${TAG_NAME}-debug` sorts a bit better and matches what `kaniko` is doing now.

Reply:
See my comment below for an alternative that I prefer, though I'm open to being convinced otherwise.

Reply:
Agreed, done.